tests/target/issue-4312.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    /* " */
+    println!("Hello, world!");
+}
```

Comment:
Would you also add some items here to cover the other snippets in https://github.com/rust-lang/rustfmt/issues/4312#issuecomment-655808193? never hurts to have more :smile:

Reply:
yeah for sure